src/nylo/interpreter.py

```diff
@@ -1,8 +1,31 @@
+# This file is a part of nylo
```

Comment:
interpreter.py should not have the copyright. Only file beggining with __ shall have them.